script/metric/import_metric_value.rb

```diff
@@ -11,6 +11,9 @@
     year = row[:year]
     value = row[:value]
     source_url = row[:source]
+    if metric_name.start_with?"EPA"
+      metric_name.gsub!("EPA","United States Environmental Protection Agency")
```

Comment:
Richard gave me a wrong metric designer name so I try to correct here before importing metric values.